drone_localization/api/endpoints/gallery.py

```
@@ -0,0 +1,316 @@
+                "br_E": request.coordinates.br_E,
+                "br_N": request.coordinates.br_N
+            }
+
+        # Загружаем через сервис
+        image_id = gallery_service.upload_image(pil_image, full_metadata)
+
+        return GalleryUploadResponse(
+            image_id=image_id,
+            status="success"
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)